src/types.deno.ts

```diff
@@ -1,5 +1,7 @@
 // === Needed imports
-import { basename } from "@std/path";
+import { basename } from "jsr:@std/path@^0.221.0";
+import { iterateReader } from "jsr:@std/io@^0.221.0";
```

Comment:
I don’t have any problems specifically with using JSR to import std modules, but I think it doesn’t make sense, at least now, since the versions of all modules there are the same.

Reply:
To be completely honest, I'm also not too comfortable with depending on yet another platform. JSR is very cool and we are going to adopt it at some point, but we currently don't use it anywhere and I would ideally like to avoid a weird transitional period where some parts use JSR and some parts don't and you never really know what's going on. This is just needless complexity. Let's do it once and do it right.

Also, we cannot use `^` because `@std` is still 0.x, so there are constantly breaking changes. We have to pin exact versions anyway.

Reply:
Yeah, you're right. I think if we use `JSR`, there's absolutely no need to publish `/x`, as `deno add` can install these packages.

Reply:
```suggestion
import { basename } from "https://deno.land/std@0.211.0/path/basename.ts";
import { iterateReader } from "https://deno.land/std@0.211.0/io/iterate_reader.ts";
```

Reply:
Thanks!